jreader/token_reader_default.go

```diff
@@ -17,6 +17,8 @@ import (
 	"unicode/utf8"
 )
 
```

Comment:
This constant and the one in `jreader/token_reader_easyjson.go` could use a comment explaining that they're used in the tests

Reply:
@cwaldren-ld I pushed an updated commit that has this comment!